### PR TITLE
Revised definitions of @api and @internal

### PR DESCRIPTION
**Pull request by mindplay-dk**
As proposed and discussed here: 

https://groups.google.com/d/msg/php-fig/tix3-i6YCQQ/Ww_uO-LCEwAJ

Note that I've removed the inline variant from the description - per it's revised definition as a visibility modifier, using it inline doesn't make any sense.

If approved, this voids #135.

### CODE REVIEW

File: proposed/phpdoc-tags.md [hunk 1]
```diff
@@ -230,38 +230,36 @@ times in each "DocBlock".
...
 
     @api
 
 #### Description
 
-The @api tag represents those "Structural Elements" with a public visibility
-which are intended to be the public API components for a library or framework.
-Other "Structural Elements" with a public visibility serve to support the
-internal structure and are not recommended to be used by the consumer.
+The `@api` tag should may applied to public "Structural Elements" to highlight
```
Comment (GaryJones) [inline proposed/phpdoc-tags.md]: The first few words do not read well - "should may applied". Maybe:

"The `@api` tag MAY be applied to..."

File: proposed/phpdoc-tags.md [hunk 2]
```diff
@@ -230,38 +230,36 @@ times in each "DocBlock".
...
+them in generated documentation, pointing the consumer to the primary public
+API components of a library or framework.
 
-The exact meaning of "Structural Elements" tagged with @api MAY differ per
-project. It is however RECOMMENDED that all tagged "Structural Elements" SHOULD
-NOT change after publication unless the new version is tagged as breaking
-Backwards Compatibility.
+Other "Structural Elements" with a public visibility may be listed less
```
Comment (GaryJones) [inline proposed/phpdoc-tags.md]: s/may/MAY ?

File: proposed/phpdoc-tags.md [hunk 3]
```diff
@@ -230,38 +230,36 @@ times in each "DocBlock".
...
+prominently in generated documentation.
 
-#### Examples
+See also the [`@internal`](#58-internal), which may be used to hide internal
```
Comment (GaryJones) [inline proposed/phpdoc-tags.md]: s/may/MAY ?

File: proposed/phpdoc-tags.md [hunk 4]
```diff
@@ -529,64 +527,41 @@ variable and a variable documented in the project.
 
 ### 5.8. @internal
 
-The @internal tag is used to denote that the associated "Structural Element" is
-a structure internal to this application or library. It may also be used inside
-a description to insert a piece of text that is only applicable for
-the developers of this software.
+The `@internal` tag is used to denote that the associated "Structural Element"
+should be regarded as being internal to the application, library or package.
+An optional description may be added to explain why.
```
Comment (GaryJones) [inline proposed/phpdoc-tags.md]: s/may/MAY ?

File: proposed/phpdoc-tags.md [hunk 5]
```diff
@@ -529,64 +527,41 @@ variable and a variable documented in the project.
...
+be regarded as exempt from the API - for example:
 
-#### Description
+  * Static analysis tools may indicate the use of internal elements from another
+    library/package with a warning or notice.
 
-The @internal tag can be used as counterpart of the @api tag, indicating that
-the associated "Structural Element" is used purely for the internal workings of
-this piece of software.
+  * Library authors may regard breaking changes to internal elements as being exempt
```
Comment (GaryJones) [inline proposed/phpdoc-tags.md]: s/may/MAY ?

File: proposed/phpdoc-tags.md (hunk 1 above)
Comment (mindplay-dk) [reply proposed/phpdoc-tags.md]: haha, not sure what happened there ;-)

**Comment by ashnazg** [comment]
Removing the inline option does take away one of the _probably irreconcilable _ points of contention regarding the double-closing-brace.  I can live with that being a "phpDocumentor feature" instead :+1:

**Comment by ashnazg** [comment]
Ping @ondrejmirtes @muglug @@neuro159 for reviews.

**Comment by muglug** [comment]
cc @erunion for opinions

**Comment by ashnazg** [comment]
@GaryJones , you still `X` on your review?

@muglug @ondrejmirtes @neuro159 , anything to add?

**Comment by ashnazg** [comment]
@mindplay-dk , please reissue these changes as a new pull request at the FIG repo itself... my apologies for having to do so.

**Comment by ashnazg** [comment]
Absorbed these changes in on https://github.com/php-fig/fig-standards/pull/1118

**Pull request closed without merging**